src/lesson-4/src/actions/createTask.ts

```diff
@@ -0,0 +1,71 @@
+import { getFormHtml } from '../components/getFormHtml';
+import { createTask } from '../server-util';
+import type { Priority, Status, Task } from '../types';
+import { updateTasks } from './updateTasks';
+
+export const setupForm = async (element: HTMLElement) => {
+  element.innerHTML = getFormHtml();
+  const item = document.querySelector<HTMLFormElement>('#simpleForm')!;
+
+  const title = item.querySelector<HTMLInputElement>('#title')!;
```

Comment:
можна одразу отримувати усі дані з форми у вигляді обʼєкту. так не треба окремо вказувати
```suggestion
const data = Object.fromEntries(new FormData(form));
const title = data.title as string 
// або так
const title = data.title.toString()
```

Reply:
так, дякую!
але чомусь мені кожного разу new FormData() повертає undefined, яку б форму я йому не вкидала :( 
тому лишила чопорний, але працюючий варіант

Reply:
бо його треба всередині `onSubmit` викликати, не на початку виконання `setupForm`
коли ти тільки викликаєш setupForm, даних ще немає
це теж робочий варіант, безперечно, але для перформансу краще зайвий раз не робити пошук по DOM